geode-core/src/test/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -109,6 +110,7 @@ public void setUp() throws Exception {
 
   @After
   public void tearDown() {
+    System.clearProperty(DistributionConfig.GEMFIRE_PREFIX + "LOCATOR_UPDATE_INTERVAL");
```

Comment:
This can be replaced with 
`@Rule
  public final RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();`